hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/transfer/AssociateTokenRecipientsStep.java

```diff
@@ -81,6 +81,14 @@ public void doIn(@NonNull final TransferContext transferContext) {
                 try {
                     newAssociation = validateAndBuildAutoAssociation(
                             accountId, tokenId, token, accountStore, tokenRelStore, handleContext);
+                    // (FUTURE) Also remove this redundant validation after diff testing
```

Comment:
Is this temp diff testing change mentioned in the PR description?  Not sure why this "redundant validation" is needed here.

Reply:
For diff-testing only.